api/base/serializers.py

```diff
@@ -353,10 +353,12 @@ def process_related_counts_parameters(self, params):
         if utils.is_truthy(params) or utils.is_falsy(params):
             return params
 
+        is_retraction = getattr(value, 'is_retracted', False)
```

Comment:
Is there a way to know whether the counted field is being hidden? A naive approach might be

```python
field.get_attribute(value) is None
```

since `HideIfRegistration`'s and `HideIfRetraction`'s get_attribute return `None` in order to hide the field. Not sure if there's a better way; haven't look too far into it.